data/requirements.txt

```diff
@@ -0,0 +1,98 @@
+apparmor==2.13.3
```

Comment:
What is this file used for?

Reply:
I think python projects are supposed to have a requirements.txt so you can just set up the dev environment by running "pip install -r requirements.txt" instead of manually installing all of the right versions of all the packages. I think I froze the wrong requirements on this version of the file because it ended up having all this extraneous stuff which my cloudtop has installed. I'll fix it so it just has the minimum requirements for the project.

Reply:
fixed it and moved it to the root folder